ui/app/AppLayouts/Onboarding2/pages/KeycardAddKeyPairPage.qml

```diff
@@ -22,7 +22,8 @@ OnboardingPage {
     states: [
         State {
             name: "inprogress"
-            when: root.addKeyPairState === Onboarding.ProgressState.InProgress
+            when: root.addKeyPairState === Onboarding.ProgressState.InProgress ||
```

Comment:
Just not to get stuck in here; this will be reworked anyway to be always "in progress" I assume